resources/lib/Library.py

```diff
@@ -250,14 +254,21 @@ def add_movie (self, title, alt_title, year, video_id, build_url):
         folder = re.sub(r'[?|$|!|:|#]',r'',alt_title)
         dirname = os.path.join(self.movie_path, folder)
         filename = os.path.join(dirname, movie_meta + '.strm')
+        progress=xbmcgui.DialogProgress()
+        progress.create(kodi_helper.get_local_string(650),movie_meta)
         if xbmcvfs.exists(filename):
             return
         if not xbmcvfs.exists(dirname):
             xbmcvfs.mkdirs(dirname)
         if self.movie_exists(title=title, year=year) == False:
+            progress.update(50)
+            time.sleep(0.5)
```

Comment:
Just curious, why are the `sleep` statements needed?

Reply:
Movies are only a single file which is written so it jumps nearly instant to 100% in dialog so it is only a soft break for better visibility.